### PR TITLE
Add Planfix Integration

### PR DESCRIPTION
**Pull request by popstas**
Adds Planfix Integration, https://planfix.ru/ (https://planfix.com). Tested on the latest versions of Google Chrome and Planfix. Inspired by #964 and Github integration.

Button appears in task header, task id added as tag.

image

### CODE REVIEW

**Comment by popstas** [comment]
Rebased to current master, fixed styles (today Planfix changed design).

File: src/scripts/content/planfix.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*jslint indent: 2 */
+/*global $: false, document: false, togglbutton: false*/
+'use strict';
+
+togglbutton.render('.taskCard-header-data-blocks .b-green-block-taskAfterLabel', {observe: true}, function (elem) {
```
Comment (langri-sha) [inline src/scripts/content/planfix.js]: The selector for `existingTag` is failing, so the button keeps getting added repeatedly. Could you change this to `.taskCard-header-data-blocks .b-green-block-taskAfterLabel:not(.toggl)` to ensure the button doesn't get added more than once.

File: src/scripts/content/planfix.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  description = titleElem.textContent;
+  if (numElem !== null) {
+    description = description.trim();
+    tags = [ numElem.textContent.replace('#', '') ];
+  }
+
+  div = document.createElement("div");
+  div.classList.add("b-toggl-btn", "toggl");
```
Comment (langri-sha) [inline src/scripts/content/planfix.js]: The style declarations use `.b-toggl-btn.planfix`. Need to match them up.

**Comment by popstas** [comment]
Updated, rebased, thanks for review!

**Comment by popstas** [comment]
@langri-sha, should I do anything else for accept PR?

**Pull request closed without merging**